iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -59,6 +75,12 @@ class ConversationMessagesCell: UICollectionViewCell {
         self.dataSource.handleEditMessage = { [unowned self] item in
             self.handleEditMessage?(item)
         }
+
+        self.dataSource.handleLoadMoreMessages = { [unowned self] cid in
+            Task {
+                self.conversationController?.loadPreviousMessages()
+            }
+        }
```

Comment:
The cell handles loading more messages. It doesn't forward the responsibility to the conversation list vc.